devito/finite_differences/differentiable.py

```diff
@@ -58,7 +58,20 @@ def is_Staggered(self):
     def _fd(self):
         return dict(ChainMap(*[getattr(i, '_fd', {}) for i in self._args_diff]))
 
-    @cached_property
```

Comment:
Just wondering why this is no longer cached?

Reply:
good point

Reply:
cached_property